common/app/model/trails.scala

```diff
@@ -24,24 +24,22 @@ object Trail {
       .orElse(elements.videos.headOption.map(_.images))
       .orElse(elements.thumbnail.map(_.images))
 
-    // Try to take the master 5:3 image. At render-time, the image resizing service will size the image according to card width.
-    // Filtering the list images here means that facia-press does not need to slim down the Trail object.
+    // Try to take the master 5:3 image, the largest and the smallest 5:3 image. At render-time, the image resizing service
+    // will size the largest image according to card width.  If the resizing service is switched off, we use the smallest.
```

Comment:
I think the comment matches the functionality I have switched it to, but you're right I should prioritise the master as it does now - so I will clean that up before this gets close to being merged.